Audiobook Player/Player/Containers/PlayerProgressViewController.swift

```diff
@@ -43,19 +43,31 @@ class PlayerProgressViewController: PlayerContainerViewController {
         return duration
     }
 
-    var tintColor: UIColor = .blue {
+    var colors: [UIColor]? {
         didSet {
-            self.progressSlider.tintColor = self.tintColor
+            guard let color = self.colors?[2] else {
```

Comment:
magic number `[2]`

Reply:
It'd be nice if we could structure somehow the array of colors returned by `CCColorCube`, so that it's clear that the first color is used by the background only and so on with the other colors

Reply:
This comment about magic numbers applies to the other uses of the array colors too like in the `PlayerMetaViewController`, it seems a bit random for that class to know that author and chapter colors are stored in the index 2 and 3 out of that array of who knows how many more elements it may have

Reply:
To be honest, all colors after index 0 are non deterministic. 
But we can put them in a struct and apply arbitrary names which align with how we use them. 

Making the values a bit less magical is definitely good 🙂